pkgs/by-name/st/streamlabels/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule (finalAttrs: {
+  pname = "streamlabels";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "teken";
+    repo = "streamlabels";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-FAu00SucWHMQbhp9mJYIdPWtbiny09LABYkeZdV10GQ=";
+  };
+
+  vendorHash = "sha256-eZQe8gstzj9BxjuZiXV1UFnuzjICJBFjiTpJNGFiyFI=";
+
+  meta = {
+    description = "A small go app to get basic stream labels for twitch";
```

```suggestion
    description = "Get basic stream labels for twitch";
```
Not start with the definite or an indefinite article.